src/main/java/com/stuypulse/robot/Robot.java

```diff
@@ -51,7 +52,9 @@ public void robotPeriodic() {
 
     /** This function is called once each time the robot enters Disabled mode. */
     @Override
-    public void disabledInit() {}
+    public void disabledInit() {    
+        // new ClimberBrakeCommand(robot.climber).schedule();
```

Comment:
- you cannot control pcm while disabled (it's in the CAN chain and that gets stopped by the rio when disabled) -- this means you just need to make sure it's running ALL the time when it's enabled because rio will turn off for you

 - if u want to be cheeky you can literally `disableBrake()` in periodic for climber because although periodic runs when disabled it doesnt do anything to the PCM (cause it's disabled)

 - another, more kosher, option is 
 
```
  public void periodic() {
     setBrake(DriverStation.isDisabled());
  }

```
but for the reasons i already mentioned even this is TECHNICALLY redundant, although clearer 

- this command in `disabledInit()` would NOT run (unless u overrode `runWhenDisabled()`) **BUT** even if that command ran, it wouldn't control the PCM at all